src/web/components/Card/Card.tsx

```diff
@@ -147,6 +151,8 @@ export const Card = ({
 	alwaysVertical,
 	minWidthInPixels,
 	dataLinkName,
+	badge,
+	badgeAlt,
```

Comment:
The linting is requiring an alt text

Reply:
I think this is worth more thought.

Alt text generally gets very little attention, I'm as guilty as anyone of seeing it as an annoyance, something that my linter is complaining about. Like most people I typically add something to make the squiggly line go away and move on. But I think we can do better here.

[This great article from the RNIB](https://www.rnib.org.uk/accessibility-guidelines-alt-text-what-you-need-know)  explains what alt text is and how it can be used.

The key things I learnt from reading this and other articles are:

Alt text is often annoying noise for people using screen readers. If it isn't adding value then you should _not_ set alt text.

If an image is key to the article, if it provides context that is relevant to the text around it, then alt text is essential to maintianing that meaning and context (alt text is required). If it is just a pretty addition with no particular relavance besides looking nice, then no alt text should be set (alt text is noise).

## What about this example here?
![Screenshot 2021-04-05 at 19 13 03](https://user-images.githubusercontent.com/1336821/113608844-f6f21400-9642-11eb-8231-df87b3c9f104.jpg)

In this particular case, we're communicating the message "Paid for by Vodafone". This is important context. Unfortunately, we're not communicating this in an accessible manner. We're assuming that everyone is sighted and can easily make the transition between text and image but that's not true and even if we do set alt text it's clunky and we're only giving the information in the card footer (long after the link was first read out and so it probably never gets accessed).

So the accessibility 'problem' that we need to solve is not what alt text should I use, but instead, it is how can we make it clear to someone using a screen reader that this content is paid for by Vodafone?

### What does good look like?
A good, accessible solution would be to have a prop called `brandName` and then to use that to create a, longer, screen reader accessible version of the 'Paid for by' text that reads a useful message: Eg. "This content was paid for by Vodafone as part of Guardian Labs". You then make this mini component hidden and only something consumed by screen readers. We do something similar for the age warning on articles and [there is a utility for this](https://theguardian.design/2a1e5182b/p/6691bb-accessibility/t/1285a8) in Source. We can then place it next to the headline so we give the information to people when they need it. If the prop `brandName` is noot provided we default to "This content has been paid for by a sponser as part of Guardian Labs".

By creating a separate mini component to read this text out we can also use the same thing when rendering the main article. Right now I'm not sure we're annoucing paid content very well to people using screen readers, it's in the article meta but sufferes from similar issues as here. We should probably place it in the same position that we do the accessible age warning, directly above or below the headline.

### Random side thought
@SiAdcock It'd maybe be useful to have a generic component for this kind of thing. The `visuallyHidden` css class is already 😍 but maybe something like 

```ts
<ScreenReaderOnly text="I am only read to screen readers" />
```

better communicates the intentions? Just a thought. It took me a while to absorb what `visuallyHidden` was about.

## Urgh, but I just want to finish my PR already!
I get it, I'm dragging this PR out, but I actually think this is a fantastic opportunity to examine what alt text is and make ourselves better developers. We care about accessibilty at the Guardian and that means that we sometimes put the extra time and effort in to create really great solutions. Should we invest this much time and effort into all code and PRs? Probably not, but hopefully by focussing in every now and then we all learn something.

Should I stop wrting this comment already? Yes. Was it too long? Possibly. Sorry.